docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -137,6 +137,54 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=include-segment-file-sizes]
 [[cluster-nodes-stats-api-response-body]]
 ==== {api-response-body-title}
 
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster.name>> setting.
+
+`nodes.<node_id>.timestamp`::
+(integer)
+Last time the node statistics were refreshed. Recorded in milliseconds since the
+https://en.wikipedia.org/wiki/Unix_time[Unix Epoch].
+
+`nodes.<node_id>.name`::
+(string)
+Human-readable identifier for the node. Based on the <<node.name>> setting.
+
+`nodes.<node_id>.transport_address`::
+(string)
+Host and port where HTTP connections for the node are accepted.
+
+`nodes.<node_id>.host`::
+(string)
+Network host for the node, based on the <<network.host>> setting.
+
+`nodes.<node_id>.ip`::
+(string)
+IP address and port for the node.
+
+`nodes.<node_id>.roles`::
+(array of strings)
+Roles assigned to the node. See <<modules-node>>.
+
+`nodes.<node_id>.attributes.ml.machine_memory`::
+(string)
+Total bytes of memory used for {ml-docs}/index.html[machine learning].
+
+`nodes.<node_id>.attributes.xpack.installed`::
+(string)
+If `true`, X-Pack features are enabled. See <<license-settings>>.
+
+`nodes.<node_id>.attributes.ml.max_open_jobs`::
+(string)
+Maximum number of {ml-docs}/index.html[machine learning] jobs that can run
+simultaneously on the node. Based on the
+<<xpack.ml.max_open_jobs,xpack.ml.max_open_jobs>> setting.
```

Comment:
Do we want to break down the attributes like this? From what I can see, these are essentially static values sourced from node configurations (`node.attr.<name>`), and may either contain more or fewer than these listed attributes.

Even on an Elastic cloud service instance, the attributes are different:

```json
"attributes" : {
  "logical_availability_zone" : "zone-0",
  "server_name" : "instance-XXXXXXXXXX",
  "availability_zone" : "us-central1-a",
  "xpack.installed" : "true",
  "instance_configuration" : "gcp.data.highio.1",
  "region" : "unknown-region"
}
```

Reply:
Thanks for the extra context. I wasn't aware it was possible to populate these differently.

Given that, I think it might be better to just document this as an object. I've done that with 382ed6b2650a91816ba913d8f2471623043c52cb.